src/main.rs

```diff
@@ -444,7 +442,8 @@ fn run_script(vm: &VirtualMachine, scope: Scope, script_file: &str) -> PyResult<
 
 #[test]
 fn test_run_script() {
-    let vm: VirtualMachine = Default::default();
+    let mut uninitialized_vm: VirtualMachine = Default::default();
+    let vm = uninitialized_vm.initialize(true);
```

Comment:
Maybe have `VirtualMachine::default()` initialize the VM and panic if it fails? That would fix small use cases like this.

Reply:
Thanks for the review.  I put the `initialize` into `default`, and new a vm, setup stdlib_inits than initialize the vm in a customized way.

Reply:
The vm will be initialized when created as default.  And the vm will be initialized after stdlib_inits changed, by putting the flag into settings.